TABTestKit/Classes/Extensions/NormalizedCoordinate+Locations.swift

```diff
@@ -0,0 +1,142 @@
+//
+//  NormalizedCoordinate+Locations.swift
+//  TABTestKit
+//
+//  Created by Kane Cheshire on 14/10/2019.
+//
+
+import XCTest
+
+public extension CGVector {
```

Comment:
NormalizedCoordinate is just a typealias of CGVector so we can just extend CGVector, so anyone using TTK can use these even if they're not using `Direction.from(to:)` above